src/Microsoft.AspNetCore.SignalR.Client.Core/HubConnection.cs

```diff
@@ -856,11 +856,14 @@ public void ResetTimeout()
 
         private async Task TimerLoop(TimerAwaitable timer)
         {
+            // tell the server you intend to ping
```

Comment:
nit: we generally use "we" to refer to what the code will do "tell the server we intend to ping"